src/kvirc/kernel/KviIrcConnectionServerInfo.cpp

```diff
@@ -275,6 +273,566 @@ KviBasicIrcServerInfo::~KviBasicIrcServerInfo()
+		case 'f': return __tr2qs("f: Recipient for full I-Line notices"); break;
+		case 'g': return __tr2qs("g: Only allow accepted clients to message you"); break;
+		case 'j': return __tr2qs("j: Recipient for rejected client notices"); break;
+		case 'l': return __tr2qs("l: Recipient for LOCOPS"); break;
+		case 'p': return __tr2qs("p: Channels hidden from WHOIS"); break;
+		case 'q': return __tr2qs("q: Idle time hidden from WHOIS"); break;
+		case 'r': return __tr2qs("r: Registered"); break;
+		case 'u': return __tr2qs("u: Recipient for unauthorized client notices"); break;
+		case 'x': return __tr2qs("x: Host hidden"); break;
+		case 'y': return __tr2qs("y: Can see stats/links/admin requests"); break;
+	}
+	return KviBasicIrcServerInfo::getUserModeDescription(mode);
+}
+
+const QString & KviIrcdSevenIrcServerInfo::getUserModeDescription(QChar mode)
+{
+	switch(mode.unicode())
+	{
+		case 'O': return __tr2qs("O: IRC help operator"); break;
+		case 'Q': return __tr2qs("Q: Prevents you from being affected by channel forwarding"); break;
+		case 'S': return __tr2qs("S: Network service"); break;
+		case 'Z': return __tr2qs("Z: Connected over SSL"); break;
+		case 'h': return __tr2qs("h: Marks you as a helper in /stats p"); break;
+		case 'p': return __tr2qs("p: Enable oper overrides"); break;
+	}
+	return KviHybridServerInfo::getUserModeDescription(mode);
+}
+
+const QString & KviPlexusIrcServerInfo::getUserModeDescription(QChar mode)
+{
+	switch(mode.unicode())
+	{
+		case 'C': return __tr2qs("C: Prevent CTCPs"); break;
+		case 'N': return __tr2qs("N: Network administrator"); break;
+		case 'U': return __tr2qs("U: Network service"); break;
+		case 'X': return __tr2qs("X: Recipient for new server introduction and split messages"); break;
+		case 'w': return __tr2qs("w: Recipient for server WALLOPS"); break;
+		case 'z': return __tr2qs("z: Recipient for oper WALLOPS"); break;
+		case 'y': return __tr2qs("y: Recipient for notices on WHOIS"); break;
+		case 'a': return __tr2qs("a: Server administrator"); break;
+		case 'q': return __tr2qs("q: Services administrator, can use SQUIT"); break;
+	}
+	return KviHybridServerInfo::getUserModeDescription(mode);
+}
+
+const QString & KviOftcIrcServerInfo::getUserModeDescription(QChar mode)
+{
+	switch(mode.unicode())
+	{
+		case 'C': return __tr2qs("C: Recipient for far connect/quit notices"); break;
+		case 'P': return __tr2qs("P: Network service"); break;
+		case 'w': return __tr2qs("w: Recipient for server WALLOPS"); break;
+		case 'z': return __tr2qs("z: Recipient for oper WALLOPS"); break;
+	}
+	return KviHybridServerInfo::getUserModeDescription(mode);
+}
+
+const QString & KviBahamutIrcServerInfo::getUserModeDescription(QChar mode)
+{
+	switch(mode.unicode())
+	{
+		case 'A': return __tr2qs("A: Server administrator"); break;
+		case 'C': return __tr2qs("C: Only accept private messages from users who share common channels"); break;
```

Comment:
```"C: Only accept private messages from users who share common channels```

thats a mouth full, I wonder if we can say this with less words.

Perhaps a feeble attempt - need way more coffee.

```"C: Only accept private messages from users in common channels```